### PR TITLE
Nilanjan test ws Nov 30

### PR DESCRIPTION
**Pull request by nil9**
Implemented test cases for first milestone

### CODE REVIEW

**Comment by RicardoUsbeck** [comment]
Please fix the codacy issues https://app.codacy.com/app/nikit91/dice-ida/pullRequest?prid=2637052

**Comment by nikit91** [comment]
Hi @nil9,
Do look into the Codacy Quality Review for your pull request as well. Any code that is merged with master should be compliant with Codacy's standards

File: ida-chatbot/src/app/service/rest/rest.service.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {Observable} from 'rxjs';
   providedIn: 'root'
 })
 export class RestService implements HttpInterceptor {
-  private hosturl = 'http://131.234.28.84:8080/ida-ws/';
+  private hosturl = 'http://localhost:8080/ida-ws/';
```
Comment (nikit91) [inline ida-chatbot/src/app/service/rest/rest.service.ts]: this shouldn't goto master

**Comment by nil9** [comment]
Codacy Issue Fixed

**Comment by nil9** [comment]
@RicardoUsbeck @nikit91 Please review the pull request. I have excluded clustering algorithms because it was throwing error by Travis ci. Once this pull request is accepted I shall create a separate branch with clustering algo and create another pull request.

File: ida-chatbot/src/app/service/rest/rest.service.ts [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-      if (prmobj[x] != null) {
-        params = params.set(x, prmobj[x]);
-      }
-    }
-    this.requestEvnt.emit(true);
-    const reqObs = this.http.get(this.getFullUrl(path), {params: params});
-    return reqObs;
-  }
-}
+import {EventEmitter, Injectable} from '@angular/core';
```
Comment (nikit91) [inline ida-chatbot/src/app/service/rest/rest.service.ts]: @nil9 this file should not be the part of commit

File: ida-ws/src/main/java/upb/ida/rest/MessageRestController.java [hunk 3]
```diff
@@ -53,7 +53,6 @@ public ResponseBean sendmessage(@RequestParam(value = "msg") String msg,
 		dataMap.put("actvDs", actvDs);
 		response.setPayload(dataMap);
 		String reply = rsService.getRSResponse(msg);
```
Comment (nikit91) [inline ida-ws/src/main/java/upb/ida/rest/MessageRestController.java]: @nil9 this file should not be the part of commit

**Pull request merged**